src/Databases/MySql.php

```diff
@@ -60,6 +63,26 @@ public function dontUseSingleTransaction()
     }
 
     /**
+     * @return $this
+     */
+    public function useSkipComments()
+    {
+        $this->useSkipComments = true;
+
+        return $this;
+    }
+
+    /**
+     * @return $this
+     */
+    public function dontUseSkipComments()
+    {
+        $this->useSkipComments = false;
```

Comment:
Could you please rename all usages of `useSkipComments` to `skipComments` and all `dontUseSkipComments` to `dontSkipComments`